lib/ansible/plugins/lookup/passwordstore.py

```diff
@@ -95,11 +100,11 @@ def parse_params(self, term):
 
     def check_pass(self):
         try:
-            self.passoutput = check_output2(["pass", self.passname]).splitlines()
+            self.passoutput = to_native(check_output2(["pass", self.passname])).splitlines()
```

Comment:
This line was changed to fix the previously mentioned Py3 errors. I'm still not entirely sure where/when it's best to convert between string/byte types.